functions/src/import-csv.ts

```diff
@@ -146,10 +146,11 @@ function csvRowToLocationOfInterest(row: any, jobId: string) {
       properties[columnName] = value;
     }
   }
-  const {latStr, lngStr, ...loi} = data;
+  const {lat: latStr, lng: lngStr, ...loi} = data;
   const lat = Number.parseFloat(latStr);
   const lng = Number.parseFloat(lngStr);
   if (isNaN(lat) || isNaN(lng)) {
+    console.error('unable to parse latitude and/or longitude values');
```

Comment:
This error will not be visible to the user, I don't the we need to log it in the backend